### PR TITLE
fix: Shorten and link if value is address

### LINKED ISSUE #274: Strategies Modal doesn't look good
**Describe the bug**
Strategies Modal doesn't look good
image

**To Reproduce**
1. Go to https://vote.cream.finance/#/cream-finance.eth/proposal/QmR4hEWEZbS5vZHBX8tsvWjXBKitVWtxVNACcHYUdBRfxc
2. Click on Strategy in the right side column
4. See an error

**Expected behavior**
maybe shorten addresses like other strategies

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: WINDOWs
 - Browser chrome
 - Version 89.0.4389.90

### PR DESCRIPTION
**Pull request by samuveth**
Fixes #274 

Changes proposed in this pull request:
- Shorten and link if value is address

image

### CODE REVIEW

File: src/components/Modal/Strategies.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
           >
             <span v-text="key" class="flex-auto text-gray mr-1" />
             <a
-              v-if="key === 'address'"
+              v-if="key === 'address' || option.length === 42"
```
Comment (bonustrack) [inline src/components/Modal/Strategies.vue:21]: How about using `isAddress` function instead here?

File: src/components/Modal/Strategies.vue [hunk 2]
```diff
@@ -44,6 +44,9 @@
 <script>
 export default {
   props: ['open', 'strategies', 'space'],
-  emits: ['close']
+  emits: ['close'],
+  mounted() {
+    console.log(this.strategies);
```
Comment (bonustrack) [inline src/components/Modal/Strategies.vue:49]: Any reason to have this?

File: src/components/Modal/Strategies.vue (hunk 1 above)
Comment (samuveth) [reply src/components/Modal/Strategies.vue]: Oh nice, added it

File: src/components/Modal/Strategies.vue [hunk 3]
```diff
@@ -42,8 +42,13 @@
...
 
 <script>
+import { isAddress } from '@ethersproject/address';
+
 export default {
   props: ['open', 'strategies', 'space'],
-  emits: ['close']
+  emits: ['close'],
+  methods: {
+    isAddress: isAddress
```
Comment (bonustrack) [inline src/components/Modal/Strategies.vue:51]:
```suggestion
    isAddress
```

**Pull request merged**